app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/components/EditWithGroups.tsx

```diff
@@ -114,10 +115,16 @@ export const EditWithGroups = () => {
 
   return (
     <>
-      <h1 className="visually-hidden">{t("edit")}</h1>
-      <div className="mb-4">
-        <SaveButton />
+      <h1 className="sr-only">{t("edit")}</h1>
+      <div className="flex w-[800px]">
+        <h2 id="questionsTitle" tabIndex={-1}>
+          {t("questions")}
+        </h2>
+        <div className="ml-5 mt-2">
```

Comment:
I ran this design by Sam. So the save button will now be beside the heading on the Questions and Translate screens